bridge/tracker/tracker.go

```diff
@@ -0,0 +1,305 @@
+package tracker
+
+import (
+	"context"
+	"encoding/json"
+	"math/big"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/umbracle/go-web3"
+)
+
+const (
+	//	required block depth for fetching events on the rootchain
+	BlockConfirmations = 6
```

Comment:
`DefaultBlockConfirmations` is more proper?

Reply:
[Renamed](https://github.com/0xPolygon/polygon-edge/pull/386/commits/d510c26dc3e711512605c997347433f79a05782e)